ShipperOptions.json

```diff
@@ -1162,7 +1166,8 @@
     "insurance": [
       {
         "display": "Discounted Insurance",
-        "value": "discounted_insurance"
+        "value": "discounted_insurance",
```

Comment:
why is this format diff than the one that's `{"discounted_insurance": {"max": N}}`?

Reply:
oh, is it like top level shipper vs at the shipping_method level?

Reply:
Yeah, rather than repeating the whole insurance nugget at the method level I just put that insurance type as the key. It's a little dirty.

Reply:
cool, it's hard to tell at the collapsed diff view sometimes.